alf/algorithms/rl_algorithm.py

```diff
@@ -662,20 +662,26 @@ def _sync_unroll(self, unroll_length: int):
         original_reward_list = []
         initial_state = self.get_initial_rollout_state(self._env.batch_size)
 
+        policy_step_time = 0.
         env_step_time = 0.
         store_exp_time = 0.
         for _ in range(unroll_length):
             policy_state = common.reset_state_if_necessary(
                 policy_state, initial_state, time_step.is_first())
             transformed_time_step, trans_state = self.transform_timestep(
                 time_step, trans_state)
-            policy_step = self.rollout_step(transformed_time_step,
-                                            policy_state)
+
+            t0 = time.time()
+            with record_time("time/_sync_unroll/1_per_rollout_step"):
```

Comment:
Actually just realized that `record_time` computes the average time spent of this code block for one training iteration. For example, if there are 100 rollout steps per iter, then `record_time` computes the sum of 100 rollout steps. So '1_per_rollout_step' name is confusing or did you actually mean to accumulate the time over rollout steps?